### PR TITLE
This fixes the id sequence problem.

### PR DESCRIPTION
**Pull request by madelman-fiksu**
We no longer have the problem for tables without primary keys

### CODE REVIEW

**Comment by dkhofer** [comment]
LGTM

**Comment by dkhofer** [comment]
Er, actually could you bump the version to 1.1.3 here? https://github.com/fiksu/bulk_data_methods/blob/master/lib/bulk_data_methods/version.rb#L2

**Comment by madelman-fiksu** [comment]
Version Bumped.

File: spec/bulk_data_methods/bulk_methods_mixin_spec.rb [hunk 1]
```diff
@@ -89,8 +94,8 @@ class Employee < ActiveRecord::Base
         Employee.create_many([{ :name => 'Keith', :company_id => 2 },
                               { :name => 'Mike', :company_id => 3 },
                               { :name => 'Alex', :company_id => 1 }])
-        Employee.all.each{ |r| r.created_at.between?(Time.now - 3.minute, Time.now + 3.minute) }.
-            should be_true
```
Comment (eslavich) [inline spec/bulk_data_methods/bulk_methods_mixin_spec.rb]: Hilarious!

**Comment by eslavich** [comment]
This LGTED as well

**Pull request merged**